### DISCUSSION TITLE
Added view and UI suggestions

### DISCUSSION THREAD

**Discussion by ai30328348** [photoprism/photoprism#4313] [Ideas]
I think the proportion of thumbnails in the div should be increased, the thumbnails should be displayed in the original scale of the image, it looks uncomfortable to use 1:1 thumbnails. Sometimes you have to open the image in order to see the top half or the bottom half of the image, this kind of 1:1 thumbnail is very unfriendly, and it wastes a lot of time for browsing the image. It is also necessary to add the ability to view the original image, even if the button will be placed in an inconspicuous place.
In addition, the background of the selected image is too faint and very similar to the background of the unselected image. Therefore, it is advisable to use colors that are easier to distinguish, including the UI.

**Comment by graciousgrey** [comment]
Thank you for your feedback :)

You can find related feature requests here:
- https://github.com/photoprism/photoprism/issues/2978
- https://github.com/photoprism/photoprism/issues/3432

**Reply by ai30328348** [reply]
Thanks for the answer, those ideas are too long and I'm having a hard time finding them
